selfdrive/car/toyota/interface.py

```diff
@@ -331,8 +324,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.longitudinalTuning.kpV = [1.3, 1.0, 0.7]
       ret.longitudinalTuning.kiBP = [0., 5., 12., 20., 27.]
       ret.longitudinalTuning.kiV = [.35, .23, .20, .17, .1]
-      ret.stoppingBrakeRate = 0.1  # reach stopping target smoothly
-      ret.startingBrakeRate = 2.0  # release brakes fast
+      ret.stoppingDecelRate = 0.4  # reach stopping target smoothly
+      ret.startingAccelRate = 8.0  # release brakes fast
```

Comment:
Why a factor of 4 here? Shouldn't it be 3 for toyota since that's the `ACCEL_SCALE`?

Reply:
Yes, good catch